packages/gitmoji-changelog-cli/src/index.js

```diff
@@ -1,20 +1,63 @@
 #! /usr/bin/env node
-
+const fs = require('fs')
 const yargs = require('yargs')
-const { logger } = require('@gitmoji-changelog/core')
+const { noop } = require('lodash')
 
-const { version } = require('../package.json')
+const { homepage } = require('../package.json')
 const { main } = require('./cli')
 
-logger.start(`gitmoji-changelog v${version}`)
+function getOutputFile({ output, format }) {
+  if (output) {
+    return output
+  }
+  if (format === 'json') {
+    return './CHANGELOG.json'
+  }
+  return './CHANGELOG.md'
+}
 
-const args = yargs
+yargs
+  .usage('Usage: $0 <command> [options]')
+  .command('$0 [release]', 'Generate changelog', (command) => {
+    command.positional('release', {
+      desc: 'Next version (from-package, next)',
+      default: 'from-package',
+    })
+  }, (argv) => {
+    const output = getOutputFile(argv)
+    const existsOuput = fs.existsSync(output)
+    main({
```

Comment:
maybe we could make a factory ?

```js
const cli = mode => argv => main({
  ...argv,
  mode,
  output: getOutputFile(argv),
})
```

So for the next two commands we can write:

```js
.command('init', 'Initialize changelog from tags', noop,  cli('init'))
```

Reply:
Nice ;)